webpack.config.babel.js

```diff
@@ -35,7 +35,10 @@ const basePlugins = [
   new webpack.NoErrorsPlugin(),
   new webpack.optimize.DedupePlugin(),
   new webpack.DefinePlugin({
-    'process.env.NODE_ENV': JSON.stringify(ENV)
+    'process.env': {
+      'NODE_ENV': JSON.stringify(ENV),
+      'ONFIDO_URL': JSON.stringify(process.env.ONFIDO_URL || 'https://apidev.onfido.com')
```

Comment:
Good point! Also because there is no way to store this variable on the backend as there is no backend... We pass the SDK url as a prop when we inject the SDK to then use it in the Capture component, I don't see the need to do this though and probably we should just return the right url based on `process.env.NODE_ENV` for both the SDK backend and API, instead of passing the prop from the ancestor to the children. What do you think?